pallets/abc-demo/src/delegate.rs

```diff
@@ -28,7 +28,18 @@ pub struct DelegateInfo {
     pub key3_rsa_pub_key: String,
 }
 
-pub fn request_single_delegate(account: AccountId32) {
+pub fn get_url(net_address: &NetAddress) -> String {
```

Comment:
It's better to return with Result<String>

Reply:
fixed